phoenix/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "phoenix-1",
```

Comment:
This needs to be `chain_name` now, and should match the value in chain.json and the directory name.

Reply:
done in b6e7a91